tfhe/src/integer/ciphertext/compact_list.rs

```diff
@@ -992,6 +992,15 @@ impl ProvenCompactCiphertextList {
     }
 }
 
+#[cfg(all(test, feature = "zk-pok"))]
+#[doc(hidden)]
+impl ProvenCompactCiphertextList {
```

Comment:
Can't this go in the impl just above ?

```rust
#[cfg(feature = "zk-pok")]
impl ProvenCompactCiphertextList {
  
   /// For testing and creating potentially invalid lists
    #[cfg(test)]
    fn infos_mut(&mut self) -> &mut Vec<DataKind> {
        &mut self.info
    }
}
```

Also, the doc hidden should not be needed as its not pub fn

Reply:
It can, I just felt like having test utils in a separate impl blocks preferrable, that’s what we did for some core primitives IIRC